### PR TITLE
six.py updated

### PR DESCRIPTION
**Pull request by helloimcarmine**
six.py updated to version 1.10.0, keeping changes of the previous version

### CODE REVIEW

**Comment by bassosimone** [comment]
Hi @helloimcarmine, could run just a quick smoke test using `examples/http_client.py` to make sure that everything is imported smoothly even after the changes and nothing breaks dramatically? Thanks.

File: examples/http_client.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 from neubot_runtime.http_client import HttpClient
 from neubot_runtime.http_message import HttpMessage
 from neubot_runtime.poller import Poller
+from neubot_runtime import compat23
```
Comment (bassosimone) [inline examples/http_client.py]: Why do you import `compat23` here? It seems you are not going to use it afterwards.

File: neubot_runtime/http_misc.py [hunk 2]
```diff
@@ -4,7 +4,10 @@
 
 ''' HTTP misc functions '''
 
-from .third_party.six import urlparse
+#from .third_party.six import moves
+#urlparse = moves.urllib.parse
```
Comment (bassosimone) [inline neubot_runtime/http_misc.py]: Could you remove the commented out lines please?

File: neubot_runtime/utils_path.py [hunk 3]
```diff
@@ -9,6 +9,8 @@
 import os
 
 from .third_party import six
```
Comment (bassosimone) [inline neubot_runtime/utils_path.py:11]: Is this import still needed here or can we get rid of it?

**Comment by bassosimone** [comment]
In addition to my previous comments:
- it seems you have also commited a .swp file by mistake. Could you remove it and also, while there, create a .gitignore in the toplevel folder to ignore `*.swp` to avoid this happening in the future?
- you committed the test file twice, could you get rid of the copy in regress?
- I prefer in this new repository that test files are in `neubot_runtime/test_foo.py` for every module foo.py that we want to test... could you move the file accordingly?

File: neubot_runtime/utils_path.py (hunk 3 above)
Comment (helloimcarmine) [inline neubot_runtime/utils_path.py:11]: No it's still important

**Comment by bassosimone** [comment]
Ok, let's merge this, than! Thanks for working on that! :+1:

**Pull request merged**